e2e/panels-suite/panelEdit_queries.spec.ts

```diff
@@ -39,12 +39,11 @@ e2e.scenario({
     e2e.components.QueryEditorRow.actionButton('Duplicate query').eq(0).should('be.visible').click();
 
     // We expect row with refId Band and A to exist and be visible
-    e2e.components.QueryEditorRows.rows().within((rows) => {
-      expect(rows.length).equals(2);
-    });
+    e2e.components.QueryEditorRows.rows().should('have.length', 2);
```

Comment:
Are the changes in these spec files related to cypress breaking changes? Should they also be mentioned or linked to in the breaking changes notice?

Reply:
They are, I'll update the description to have a link for the cypres changelog.